### PR TITLE
[quest] ADDED: 'Carve' Hunter Fake Rune Quests To QuestDB

### LINKED ISSUE #5603: [quest] 'Carve' Hunter Rune Quest Missing From QuestDB
## Description

The SoD Hunter rune quest 'Carve' is not currently in the QuestDB. 

## Wowhead link

https://www.wowhead.com/classic/guide/season-of-discovery/classes/hunter/carve-rune
https://www.wowhead.com/classic/spell=425758/engrave-gloves-carve

## Screenshots

N/a

## Questie & WoW version

v9.2.17
WoW Classic SoD - v1.15.052409

### PR DESCRIPTION
**Pull request by Sommos**
## Issue references

Fixes #5603
Linked To #5443

## Proposed changes

- ADDED: All 'Carve' Hunter fake rune quests and they are now in the QuestDB, they are currently hidden due to spawns of associated NPCs.

### CODE REVIEW

File: Database/Corrections/sodQuestFixes.lua [hunk 1]
```diff
@@ -3180,6 +3180,54 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -410095,
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [90206] = {
+            [questKeys.name] = "Carve",
+            [questKeys.startedBy] = {{1115,1116,1117,1118,208711}},
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:3185]: Spawns of `208711` should be cleaned up.

File: Database/Corrections/sodQuestFixes.lua [hunk 2]
```diff
@@ -3180,6 +3180,54 @@ function SeasonOfDiscovery:LoadQuests()
...
+            [questKeys.questLevel] = 9,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.HUNTER,
+            [questKeys.objectivesText] = {"Kill Rockjaw Troggs until Rabbit Musk drops, consume it and use Tame Beast on a Rabbit critter, then speak to Toby at Amberstill Ranch to receive the rune."},
+            [questKeys.requiredSpell] = -425758,
+            [questKeys.zoneOrSort] = sortKeys.HUNTER,
+        },
+        [90207] = {
+            [questKeys.name] = "Carve",
+            [questKeys.startedBy] = {{2009,2010,2011,2012,2013,2014,2152,209948}},
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:3197]: Same for `209948`

File: Database/Corrections/sodQuestFixes.lua [hunk 3]
```diff
@@ -3180,6 +3180,54 @@ function SeasonOfDiscovery:LoadQuests()
...
+        },
+        [90208] = {
+            [questKeys.name] = "Carve",
+            [questKeys.startedBy] = {{3111,3114,3113,3112,5824,5826,208184}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 8,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.HUNTER,
+            [questKeys.objectivesText] = {"Kill Razermane Quillboars until Adder Pheromone drops, consume it and use Tame Beast on a Adder critter, then speak to Razzil in Razor Hill to receive the rune."},
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:3215]:
```suggestion
            [questKeys.objectivesText] = {"Kill Razermane Quillboars until Adder Pheromone drops, consume it and use Tame Beast on an Adder critter, then speak to Razzil in Razor Hill to receive the rune."},
```

File: Database/Corrections/sodQuestFixes.lua [hunk 4]
```diff
@@ -3180,6 +3180,54 @@ function SeasonOfDiscovery:LoadQuests()
...
+            [questKeys.requiredSpell] = -425758,
+            [questKeys.zoneOrSort] = sortKeys.HUNTER,
+        },
+        [90209] = {
+            [questKeys.name] = "Carve",
+            [questKeys.startedBy] = {{2949,2950,2951,207515,205635}},
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:3221]: We should also cleanup the spawns of `205635` and not show [Lurkmane](https://www.wowhead.com/classic/npc=207515/lurkmane) as a source of the musk.

```suggestion
            [questKeys.startedBy] = {{2949,2950,2951,205635}},
```

File: Database/Corrections/sodQuestFixes.lua (hunk 1 above)
Comment (Sommos) [reply Database/Corrections/sodQuestFixes.lua:3185]: Added in: https://github.com/Questie/Questie/pull/5608/commits/f3e8eb78740cd80a1e9962c386390fafc53a7857

File: Database/Corrections/sodQuestFixes.lua (hunk 3 above)
Comment (Sommos) [reply Database/Corrections/sodQuestFixes.lua]: Fixed in: https://github.com/Questie/Questie/pull/5608/commits/b6a6590f6bdaf0f12a1a767d8e980d573aefa508

File: Database/Corrections/sodQuestFixes.lua (hunk 2 above)
Comment (Sommos) [reply Database/Corrections/sodQuestFixes.lua:3197]: Fixed in: https://github.com/Questie/Questie/pull/5608/commits/b6a6590f6bdaf0f12a1a767d8e980d573aefa508

**Review by BreakBB — APPROVED** [review]

**Pull request merged**